docs/content/_navigation.json

```diff
@@ -814,6 +814,10 @@
               {
                 "title": "Pythonic resources",
                 "path": "/guides/dagster/pythonic-resources"
+              },
+              {
+                "title": "Migrating to Pythonic resouces and config",
+                "path": "/guides/dagster/migrating-to-pythonic-resources-and-config"
```

Comment:
should this belong to "General" guides rather than "Experimental features"?

Reply:
yup.